packages/font/package.json

```
@@ -20,7 +20,11 @@
     "url": "git@github.com:nl-design-system/rijkshuisstijl-community.git",
     "directory": "packages/font"
   },
+  "scripts": {
+    "build:css": "sass --pkg-importer=node src/index.scss dist/index.css"
```

Aantal puntjes:
- Voeg build script toe, zodat wanneer build word geroepen op workspace niveau, deze build ook uitvoert.
- Op dit moment genereert het ook een source map, die kunnen we weglaten gezien dat alleen voor development is.
- De output is nog niet compressed.
```suggestion
    "build": "npm-run-all build:**",
    "build:css": "sass --load-path=node_modules --no-source-map --style=compressed src/index.scss dist/index.css"
```

---

Top, resolved!